source/usr/lib/mcvirt/config_file.py

```diff
@@ -65,7 +66,7 @@ def _writeJSON(data, file_name):
         config_file.close()
 
         # Check file permissions, only giving read/write access to libvirt-qemu/root
-        os.chmod(file_name, 0600)
+        os.chmod(file_name, 0o600)
```

Comment:
As discussed, this change is a little worrying (as it was automatically changed by autopep8). Although, this line is checked by the system tests